app/src/main/java/de/westnordost/streetcomplete/screens/settings/questselection/QuestSelectionAdapter.kt

```diff
@@ -303,7 +319,7 @@ class QuestSelectionAdapter(
                 AlertDialog.Builder(compoundButton.context)
                     .setTitle(R.string.enable_quest_confirmation_title)
                     .setMessage(item.questType.defaultDisabledMessage)
-                    .setPositiveButton(android.R.string.ok) { _, _ -> applyChecked(b) }
+                    .setPositiveButton(android.R.string.ok) { _, _ -> applyChecked(true) }
```

Comment:
What's this?

Reply:
That was a minor IDE lint warning that I fixed while editing the file: The variable `b` is guaranteed to be true here, since we explicitly check for `!b` a few lines above.

Reply:
I was never convinced by such lint rule - that is asking for bugs if b will stop being checked, while for now is not really causing any problems

Reply:
Well, if we change the condition above, we have to change the "ok" and "cancel" button behavior anyway, so I don't think we have to prepare for that. And `applyCheck(true)` is much easier to understand than `applyCheck(!b)` (What is the meaning of `b`? And what is it's current value when clicking the button?).

Reply:
I guess that it is matter of preferences, and hard to prove superiority...

Note that it is `applyChecked(b)` not `applyChecked(!b)`

also, it makes it symmetric to `applyChecked(b)` above

Reply:
Could you revert it to as it was before?

Reply:
Restored in [`0d90dfb`](https://github.com/streetcomplete/StreetComplete/pull/5284/commits/0d90dfbc279b4b7a8a055b88b872b68e4d861b19).